Dockerfile

```diff
@@ -27,5 +27,8 @@ RUN set -ex; \
 
 EXPOSE 8080
 
+RUN useradd courier_user
```

Comment:
Think you can just call the user `courier`. Also create a group. Here's an example:
https://github.com/praekeltfoundation/docker-django-bootstrap/blob/develop/Dockerfile#L8-L9 (although you don't need to specify the uid/gid).